lib/elixir_boilerplate_graphql/schema.ex

```diff
@@ -10,8 +10,8 @@ defmodule ElixirBoilerplateGraphQL.Schema do
     import_fields(:application_queries)
   end
 
-  # Even if having an empty mutation block is valid and works in Ansinthe, it
-  # causes a Javascript error in GraphiQL so uncomment it when you add the
```

Comment:
Is GraphiQL still included in the boilerplate?

Reply:
No, but still, we don’t want to generate an invalid schema for external tools that could use our GraphQL endpoint.